docs/core/index.md

```diff
@@ -13,15 +13,15 @@ See [About .NET Core](about.md) to learn more about .NET Core, including its cha
 
 Check out [.NET Core Tutorials](tutorials/index.md) to learn how to create a simple .NET Core application. It only takes a few minutes to get your first app up and running. If you want to try .NET Core in your browser, look at the [Numbers in C#](../csharp/tutorials/intro-to-csharp/numbers-in-csharp.yml) online tutorial.
 
-## Download .NET Core 2.2
+## Download .NET Core
```

Comment:
Good call

Reply:
Thanks, I thought it would be better not to call out any specific version.